src/components/Menus/Sidebar/ProjectsPanel/ProjectsPanel.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Button from "../../../Button/Button";
+import SidebarPanel from "../SidebarPanel";
+import { useTranslation } from "react-i18next";
+
+import "./ProjectsPanel.scss";
+import ProjectName from "../../../ProjectName/ProjectName";
+import ProjectInfo from "./ProjectInfo/ProjectInfo";
+import { DownloadIcon } from "../../../../Icons";
+import DownloadButton from "../../../DownloadButton/DownloadButton";
+import { useSelector } from "react-redux";
+
+const ProjectsPanel = () => {
+  const {
+    t,
+    i18n: { language: locale },
+  } = useTranslation();
+
+  const isLoggedIn = useSelector((state) => state?.auth?.user);
+
+  return (
+    <SidebarPanel
+      heading={t("projectsPanel.projects")}
+      Button={() =>
+        isLoggedIn ? (
+          <Button
+            text={t("projectsPanel.yourProjectsButton")}
+            className="btn--primary projects-panel__your-projects-button"
+            buttonIconPosition="right"
+            href={`/${locale}/projects`}
+          />
```

Comment:
@PetarSimonovic that's what these lines are, but me and Dan talked it through and it only made sense when logged in (ie they've got projects to view!)